dashboard/app/helpers/plc/learning_modules_helper.rb

```diff
@@ -3,10 +3,13 @@ def options_for_plc_task_learning_modules
     Plc::LearningModule.all.pluck(:name, :id).sort
   end
 
-  def options_for_evaluation_answer_modules(course_unit)
-    course_unit.plc_learning_modules.order(:required, :name).pluck(:id, :required, :name).map do |id, required, name|
-      # Boo Ruby, you can't add nil to a string without choking?
-      ["#{name}#{' - Required' if required}", id]
+  def options_with_learning_module_names_and_required_tag(course_unit)
+    course_unit.plc_learning_modules.order(:required, :name).map do |learning_module|
+      [learning_module.get_name_with_required_tag, learning_module.id]
     end
   end
+
+  def learning_modules_select_size options
+    [options.size, 25].min
+  end
```

Comment:
Unifying the last two is doable. Unifying the first one with them is problematic, since it's not part of a form for an object